convert2rhel/main.py

```diff
@@ -219,6 +219,10 @@ def post_ponr_conversion():
     pkghandler.replace_non_red_hat_packages()
     loggerinst.task("Convert: List remaining non-Red Hat packages")
     pkghandler.list_non_red_hat_pkgs_left()
+    
+    loggerinst.task("Convert: Patch yum configuration file")
+    redhatrelease.YumConf().patch()
```

Comment:
Since you moved the patching here, you should remove the patching from the `pre_ponr_conversion` function.

Also, as described in the PR description, the config file does not need to be backed up and restored anymore. That means:

- remove `redhatrelease.yum_conf.backup()` from `main()`
- remove `redhatrelease.yum_conf.restore()` from `rollback_changes()`
- remove `yum_conf = utils.RestorableFile(YumConf.get_yum_conf_filepath())  # pylint: disable=C0103` from `redhatrelease.py `